### ISSUE TITLE
Inconsistency between crazyflie 2.0 datasheet and code relating to N_IO_1 pin and WKUP pin

### ISSUE THREAD

**Issue by trakevital** [bitcraze/crazyflie2-nrf-firmware#66]
As you will see below, the N_IO_1 pin which should be at pin 11 according to the datasheet, is acutally the WKUP_PIN in the firmware code, while pin 12 which should be the WKUP_PIN is available, which I am guessing means that it is N_IO_1.

![image](https://user-images.githubusercontent.com/32649570/136943095-098ee057-0863-4b01-9743-ef24fc5cb36a.png)
 ``` c
  #define PM_VCCEN_PIN 10
  #define BUTTON_PIN 17

  #define PM_VBAT_PIN 1
  #define PM_VBAT_SINK_PIN 2
  #define PM_PGOOD_PIN 4

  #define USE_OW_IO_PULLUP
  #define OW_PULLUP_PIN 9
  #define OW_PIN 8
  #define WKUP_PIN 11

  #define LED_PIN 13

  #define PM_CHG_PIN 0
  #define AIN_VBAT ADC_CONFIG_PSEL_AnalogInput2
  #define AIN_ISET ADC_CONFIG_PSEL_AnalogInput4
```
Please tell me if I am simply misunderstanding something, but if I truly found an inconsistency in the documentation then I hope it will be fixed in the future in order to avoid confusion.

Sources:
* [link to lines of code](https://github.com/bitcraze/crazyflie2-nrf-firmware/blob/master/interface/pinout.h#L49-L67)
* [link to datasheet](https://www.bitcraze.io/documentation/hardware/crazyflie_2_0/crazyflie_2_0-datasheet.pdf)

**Comment by tobbeanton** [comment]
Thanks for finding this and I can confirm the issue. The drawing is wrong and P0.11 and P0.12 should swap places.

**Comment by gemenerik** [comment]
Fixed in bitcraze/hardware#13, bitcraze/hardware#14, #67
